build.sbt

```diff
@@ -297,8 +297,8 @@ lazy val finatraServerCats: Project =
       name := "tapir-finatra-server-cats",
       libraryDependencies ++= Seq(
         "org.typelevel" %% "cats-effect" % Versions.cats,
-        "io.catbird" %% "catbird-finagle" % Versions.finatra,
-        "io.catbird" %% "catbird-effect" % Versions.finatra
+        "io.catbird" %% "catbird-finagle" % "19.11.0",
```

Comment:
shouldn't this be the same version as finatra?

Reply:
I checked the version of Catbird, the latest version is `19.11.0.` I looked at its build.sbt and found the master branch actually uses Finatra 19.12.0. However, I am not sure when Catbird will bump its version to 19.12.0. :-) I used the PR build in my test project and everything worked, thus submitted this PR. Or we can wait for another couple more days to see if Catbird will release version 19.12.0?

Reply:
Ok, I have no idea if these two are bound or not :) If you didn't spot any problems, I suppose that's fine. Maybe we can then define a new version for `catbird`, as the two entries seem to be the same?

Reply:
Yes, certainly! Pardon that I did not think of that earlier... :D Shall do!